src/include/smf/rpc_recv_context.h

```diff
@@ -26,7 +27,7 @@ struct rpc_recv_context {
   static seastar::future<stdx::optional<rpc_recv_context>> parse_payload(
     rpc_connection *conn, rpc::header hdr);
 
-  rpc_recv_context(rpc::header hdr, seastar::temporary_buffer<char> body);
+  rpc_recv_context(const seastar::socket_address sa, rpc::header hdr, seastar::temporary_buffer<char> body);
```

Comment:
s/sa/remote_address/g

also please implement the move constructor: 

https://github.com/senior7515/smf/blob/master/src/core/rpc_recv_context.cc#L29

Reply:
Weird. I actually have that implemented in my branch. Not sure why it doesn't show up in this diff...